operator/pkg/reconciliation/construct_service.go

```diff
@@ -0,0 +1,197 @@
+// Copyright DataStax, Inc.
```

Comment:
There should be no changes to the logic in this file.  I simply moved service related functions to a new file.